panacea/src/pml_lexer.xrl

```diff
@@ -1,16 +1,22 @@
 Definitions.
 
-IDENT = [a-zA-Z0-9_]+
-WS    = [\s\t\n\r]+
-NUMBER = ([0-9]+(\.[0-9]+)?|\.[0-9]+)([eE][+-]?[0-9]+)?
-STRING = "[^"]*"
+IDENT   = [a-zA-Z0-9_]+
+WS      = [\s\t\n\r]+
+NATURAL = [0-9]+
+NUMBER  = ({NATURAL}(\.{NATURAL})?|\.{NATURAL})([eE][+-]?{NATURAL})?
```

Comment:
What do you think about changing `NUMBER` to something like `FLOAT`?
At the moment it still matches natural numbers as well as floats - seeing as we now have `NATURAL` we could remove that bit of the regex:

```
FLOAT  = ({NATURAL}\.{NATURAL}|\.{NATURAL})([eE][+-]?{NATURAL})?
```

something like the above.